pkgs/tools/admin/otc-auth/default.nix

```
@@ -0,0 +1,31 @@
+{ pkgs, lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
+  pname = "otc-auth";
+  version = "2.0.3";
+
+  src = pkgs.fetchFromGitHub {
+    rev = "v${version}";
+    owner = "iits-consulting";
+    repo = "otc-auth";
+    sha256 = "sha256-dKBsceTv4SW3YWuVtC7U4u0IY9VZOrJ+QM5Jq+DW4Hw=";
+  };
+  vendorHash = "sha256-J2Xo5HCycc0FU5nMfa9eDwtL94fF4/4x4xxR+Fonk/E=";
+  HOME = "/tmp";
+
+  nativeBuildInputs = [ pkgs.installShellFiles ];
+  postInstall = ''
+    installShellCompletion --cmd otc-auth \
+        --bash <($out/bin/otc-auth completion bash) \
+        --fish <($out/bin/otc-auth completion fish) \
+        --zsh <($out/bin/otc-auth completion zsh)
+  '';
+
+  meta = with lib; {
+    homepage = "https://github.com/iits-consulting/otc-auth";
+    description = "Open Source CLI for the Open Telekom Cloud written in go.";
+    license = licenses.gpl3;
```

```suggestion
    license = licenses.gpl3Only;
```

https://github.com/NixOS/nixpkgs/blob/ff67dad5711efe6daf0b1a0038b17093617ae502/lib/licenses.nix#L1156